### PR TITLE
BaseClient Discord snap path

### PR DESCRIPTION
**Pull request by BigHeadGeorge**
Same changes as #40 
Ran the test file manually and it worked :)
When I use tox it has trouble importing pypresence for some reason idk why

### CODE REVIEW

**Comment by LewdNeko** [comment]
Not gonnya lie, never even heard of snap. Kinda just makes me think ??? about this PR.

**Comment by qwertyquerty** [comment]
If a single other person also wants this change, we will merge.

**Comment by LewdNeko** [comment]
Correction: no.

**Comment by GhostofGoes** [comment]
Snaps (aka Snapcraft) are the up and coming method of distribution for user apps on Linux. Ubuntu 18 is making heavy use of them, which has sped adoption. In fact Discord is in the "editor picks" section on the front page of the Ubuntu store now. The big advantage of them over packages is they're self-contained and sandboxed, so they can be safely updated independent of the rest of the system, much like a Android app or Windows UWP app.

Long story short: going to see more people on Linux using discord through Snaps, so if this helps us support that, I support it.

**Comment by LewdNeko** [comment]
too tired to read whatever you said, so ok
merged
